### PR TITLE
La 384 cleanups

### PR DESCRIPTION
**Pull request by evrardjp**
Issue: [LA-384](https://rpc-openstack.atlassian.net/browse/LA-384)

### CODE REVIEW

**Comment by evrardjp** [comment]
recheck_cit_all

**Comment by evrardjp** [comment]
recheck_cit_artifact

**Comment by mancdaz** [comment]
@evrardjp seems the removal of -x is causing issues in Travis

**Comment by evrardjp** [comment]
why??

**Comment by evrardjp** [comment]
I guess it's that we are using a too old version of reno, with our weird 'r' release names that don't go well with reno.

**Comment by evrardjp** [comment]
recheck_cit_artifact

**Comment by evrardjp** [comment]
recheck_cit_trustry_leapfrogupgrade_swift

**Comment by evrardjp** [comment]
rebased, this way travis updates its refs.

File: scripts/leapfrog/README.md [hunk 1]
```diff
@@ -107,6 +107,8 @@ If a leapfrog upgrade is interrupted, it can be resumed. Each major step of the
 leapfrog upgrade process creates a marker file, which will be used to skip
 completed tasks on subsequent runs.
 
+To resume, run scripts/leapfrog/ubuntu14-leapfrog.sh.
```
Comment (major) [inline scripts/leapfrog/README.md]: Would be nice to have backticks for the script name, but I'm not holding it back for that. ;)

**Pull request merged**